Recipe/recipe.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title> My Static Recipe Page</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Sarah's Recipe Page</title>
+  <link href="./recipe.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+ <h1 class="center" style="margin-bottom: 0"><b>Imbaattan Recipe</b></h1>
+ <h4 class="center"> (Fried Potato Wedges with Minced Meat) </h4>
+  <script src="script.js"></script>
+  
+<div class="recipe-image">
+<img src="https://arousingappetites.com/wp-content/uploads/2022/08/MBattan.jpeg?ezimgfmt=ng%3Awebp%2Fngcb1%2Frs%3Adevice%2Frscb1-1" width="250" height="200" class="center" style="margin-bottom: 50px">
+</div>
+<p class="center"><b>Description:</b>This delcious deep fried finger food originates and is unique to Libya, my home country in North Africa! It's especially popular during special occasions as well as Ramadan. The well seasoned flavours of the meat and the combination of the crispy exterior of the potatoes might make this your new favourite side dish! North African recipes are passed down generationally and typically don't have exact measurments. Below is the closest appoximation of measurments to the recipe my mother makes.<p class="center">Enjoy and Sahtain! (Arabic word for Bon Appetit!) </p>   </p>
+<img src="https://www.worldatlas.com/r/w960-q80/upload/91/07/a8/shutterstock-751043653.jpg" width="260" height="210" class="center">
+
+<div>
+    <h4 class="center">Meat mixture:</h4>
+    <ul class="centerforText">
+        <li>1.5 lb ground beef</li>
+        <li>5-6 large potatoes</li>
+        <li>1 cup finely chopped parlsey</li>
+        <li>1/2 cup finely chopped cilantro</li>
+        <li>1 medium onion finely chopped</li>
+        <li>2 cloves garlic grated</li>
+        <li>1 tablespoon salt</li>
+        <li>2 egg whites</li>
+        <li>Preferably use Libyan Spice Mix if available otherwise add:</li>
+        <li>1/4 teaspoon black pepper</li>
+        <li>1/2 teaspoon ginger</li>
+        <li>a few pinches of cinnamon</li>
+        <li>1/4 teaspoon tumeric</li>
+        <h4 class="center">Coating:</h4>
+        <li>4 tablespoons fine breadcrumbs</li>
+        <li>2 large beaten egg yolks</li>
+        <li>Canola oil for frying</li>
```

Comment:
When you're inside of `<ol>` or `<ul>`, you should only have `<li>` items. In this case, you don't want an `<h4>` in here. You could do something like this instead,

```html
<ul>
  ...
  <li>1/4 teaspoon tumeric</li>
  <li>
    <b>Coating:</b>
    <ul>
      <li>4 tablespoons fine breadcrumbs</li>
      ...
    </ul>
  </li>
  ...
</ul>
```